tests/foreman/cli/test_leapp_client.py

```diff
@@ -0,0 +1,276 @@
+"""Tests for leapp upgrade of content hosts with Satellite
+
+:Requirement: leapp
+
+:CaseLevel: Integration
+
+:CaseComponent: LeappIntegration
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+:Upstream: No
+"""
+import pytest
+from wait_for import wait_for
+
+from robottelo.cli.repository import Repository
+from robottelo.constants import PRDS
+
+RHEL_REPOS = {
+    'rhel7_9_server': {
+        'id': 'rhel-7-server-rpms',
+        'name': 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7.9',
+        'releasever': '7.9',
+        'reposet': 'Red Hat Enterprise Linux 7 Server (RPMs)',
+        'product': 'Red Hat Enterprise Linux Server',
+    },
+    'rhel7_server_extras': {
+        'id': 'rhel-7-server-extras-rpms',
+        'name': 'Red Hat Enterprise Linux 7 Server - Extras RPMs x86_64',
+        'releasever': '7',
+        'reposet': 'Red Hat Enterprise Linux 7 Server - Extras (RPMs)',
+        'product': 'Red Hat Enterprise Linux Server',
+    },
```

Comment:
**Note:** I added `rhel7` repo data for testing purposes and this will be used in another upgrade path in future, so the review is not needed for these lines ;)